nero_view.py

```diff
@@ -83,6 +85,12 @@ def update_buttons(self):
             end_time = time.time()
             pass
         self.after(1, self.update_buttons)
+    
+    def update_pinned_data(self):
+        if time.time() - self.last_pinned_update_time >= 1:
+            self.model.update_pinned_data()
+            self.last_pinned_update_time = time.time()
+        self.after(1, self.update_pinned_data)
```

Comment:
so we're calling this function every one millisecond but only want to update every 1 total second? maybe we should call it after 100 instead just for efficeincy

Reply:
yeah we can do that